src/Context/ContextProvider.jsx

```diff
@@ -1,13 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
+import axios from 'axios';
 import UserContext from './UserContext';
-import capitalsData from '../common/data';
 
 function ContextProvider({ children }) {
-  // capitals will come from request from Freca Api
-  const capitals = capitalsData;
+  const [token, setToken] = useState('');
 
-  return <UserContext.Provider value={{ capitals }}>{children}</UserContext.Provider>;
+  useEffect(async () => {
```

Comment:
Are there any warnings in console regarding to such way of declaring effect function?

Reply:
No I don't have warnings in console!